gimme

```diff
@@ -182,6 +183,21 @@ _compile() {
 	)
 }
 
+_can_compile() {
+	(
+		mkdir -p "${GIMME_TMP}/compile-test"
+		cd "${GIMME_TMP}/compile-test"
+		cat > main.go <<EOF
+package main
+import "fmt"
+func main() {
+	fmt.Println("compile test")
+}
+EOF
+		"${1}/bin/go" get -v ./...
```

Comment:
Why not just `"${1}/bin/go" build -v`, since we don't have (or want) anything to actually be downloaded?

You could even skip the whole directory if you wanted to get sneaky, with something like:
```bash
_can_compile() {
	cat > "${GIMME_TMP}/test.go" <<'EOF'
package main
import "os"
func main() {
	os.Exit(0)
}
EOF
	"${1}/bin/go" run "${GIMME_TMP}/test.go"
}